awx/ui_next/src/screens/Host/HostEdit/HostEdit.jsx

```diff
@@ -31,6 +35,10 @@ function HostEdit({ host }) {
       await HostsAPI.update(host.id, values);
       history.push(detailsUrl);
     } catch (error) {
+      // check for field-specific errors from API
+      if (error.response?.data && typeof error.response.data === 'object') {
+        throw error.response.data;
```

Comment:
This is probably where you'd want to start implementing the error handling code for field errors.

This seems like as good a place as any to get started, how do you feel about using this PR as a staging ground for that work? The next step would be to extend it to the job templates form, and then the ~world~ rest of the app.

Reply:
So, this grabs field errors out of the response, if present, and throws them. Inside the form component, that gets caught and passed into Formik. Field-specific errors don't really need to be handled other than that, since they come back from the API in exactly the format Formik needs (via `formik.setErrors()`)

Reply:
I am working on WFJT Form now and I was considering following this pattern here for error handling on that form.